dist/tools/firmware_metadata/README.md

```diff
@@ -0,0 +1,33 @@
+# Metadata generator for OTA firmware updates
+This program will generate a binary file containing a metadata structure as
+follows:
+
+```c
+typedef struct FW_metadata {
```

Comment:
It doesn't matter if it's packed as the metadata will always be read by the same MCU. But it's nicer and probably more compact :-)